_posts/2020-07-07-aws-localstack.md

```diff
@@ -0,0 +1,252 @@
+---
+title: "Local development with AWS on LocalStack"
+categories: [craft]
+date: 2020-06-30 06:00:00 +1100
+modified: 2020-06-30 06:00:00 +1100
+author: pratikdas
+excerpt: "This article describes LocalStack as a useful aid to test your AWS services locally."
+image:
+  auto: 0072-aws
+---
+During the initial days of development, we prefer to focus on writing code for our application instead of spending time on setting up the environment for accessing AWS services. We access various AWS services while building our applications - for example, upload files in S3, store some data in DynamoDb, send message to SQS, etc.  
+
+***Setting up a development environment for using these services is complex and time-consuming. Instead, we use [LocalStack](https://github.com/localstack/localstack) to develop and unit test our applications with mock implementations of these services.*** We switch to the real services only in the integration environment and beyond.
+
+ {% include github-project.html url="https://github.com/pratikdas/localstack" %}
```

Comment:
Please raise a PR in the code-examples repository (https://github.com/thombergs/code-examples/pulls) and add the code to a module `aws/localstack` there. Then, please update the links in this article.